README.md

```diff
@@ -26,6 +26,19 @@ You can build *kafka-connect-storage-cloud* with Maven using the standard lifecy
 - Source Code: https://github.com/confluentinc/kafka-connect-storage-cloud
 - Issue Tracker: https://github.com/confluentinc/kafka-connect-storage-cloud/issues
 
+# Running Integration Tests
+
+In order to run integration tests, AWS credential of the bucket owner and an addition user.
+
+Add the following Environmental variables :
+
+SECONDARY_USER_ACCOUNT_ID=********;
+SECONDARY_USER_ACCESS_KEY_ID=********;
+SECONDARY_USER_SECRET_ACCESS_KEY=********;
+ROOT_USER_ACCESS_KEY_ID=********;
+ROOT_USER_SECRET_ACCESS_KEY=********;
```

Comment:
Surround these with 3 back-ticks(`) to make them render at code snippet in individual lines. Currently, they are all in the same line.

Reply:
Also, since we this is something exclusive to S3 tests, let's mention it on S3 Readme.
Also, the path in the javadocs mentioned in IT have to be changed -> `/kafka-connect-s3/REAMDE.md`

Reply:
> Surround these with 3 back-ticks(`) to make them render at code snippet in individual lines. Currently, they are all in the same line.

done